### PR TITLE
Add unit test for Social component

### LINKED ISSUE #301: add unit test cases for social component
**Description**
Add a unit test case for the `social` component

**Describe the solution you'd like**
You can get an idea from the test case written for the `title` component.
```js
import React from "react"
import renderer from "react-test-renderer"

import Title from "../title"

describe("Title", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Title prefix={{ title: "Hi 👋, I'm" }} data={{ title: "" }} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
```

**Note**
Add your unit tests in `src/components/__tests__`

Join the **Discord Server** for further discussions.
<a href="https://discord.gg/HHMs7Eg">
GPRG Discord Server Link
</a>
Server Link: https://discord.gg/HHMs7Eg

### PR DESCRIPTION
**Pull request by Tekipeps**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Enhancement
- [ ] Documentation Update

## Description
Add render test for Social  #component

## Related Tickets & Documents

fixes #301

### CODE REVIEW

File: src/components/__tests__/social.test.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from "react"
+import renderer from "react-test-renderer"
```
Comment (Citrinin) [inline src/components/__tests__/social.test.js:2]: Thanks for your efforts, looks good!
But we shall rely only on one testing library use `enzyme-to-json` library and do not use `react-test-renderer`  - there is no trace of it in `master` now. How to work with it you can see [here](https://github.com/rahuldkjain/github-profile-readme-generator/blob/master/src/components/__tests__/work.test.js#L22)

Comment (Tekipeps) [reply src/components/__tests__/social.test.js:2]: Thank you, I will continue to work on it

Comment (Tekipeps) [reply src/components/__tests__/social.test.js]: I have written the test and it now passes fine
image

**Comment by Tekipeps** [comment]
I have written the test with enzyme and ran the format script

image

thank you

**Comment by rahuldkjain** [comment]
Hey @Tekipeps, please unstage all the changes which are not relevant to #301

**Comment by Tekipeps** [comment]
@rahuldkjain fixed it sir

**Review by Citrinin — APPROVED** [review]

**Pull request merged**